tests/phpunit/tests/blocks/wpGetBlockPatterns.php

```diff
@@ -0,0 +1,129 @@
+<?php
+/**
+ * Tests for _wp_get_block_patterns.
+ *
+ * @package WordPress
+ * @subpackage Blocks
+ * @since 6.4.0
+ *
+ * @group blocks
+ *
+ * @covers ::_wp_get_block_patterns
+ */
+class Tests_Blocks_wpGetBlockPatterns extends WP_UnitTestCase {
```

Comment:
```suggestion
class Tests_Blocks_WpGetBlockPatterns extends WP_UnitTestCase {
```
Even though there's examples of `wp` in the tests, these actually violate [the standard on test class names](https://make.wordpress.org/core/handbook/testing/automated-testing/writing-phpunit-tests/#test-classes) - Should always be TitleCase.